operate/client/e2e-playwright/api/camunda8.ts

```diff
@@ -11,7 +11,8 @@ import {Camunda8} from '@camunda8/sdk';
 const camunda8 = new Camunda8({
   CAMUNDA_OAUTH_DISABLED: true,
   CAMUNDA_SECURE_CONNECTION: false,
-  ZEEBE_REST_ADDRESS: 'http://localhost:8086',
+  ZEEBE_REST_ADDRESS:
+    process.env.ZEEBE_REST_GATEWAY_ADDRESS || 'http://localhost:8086',
```

Comment:
```suggestion
    process.env.ZEEBE_REST_GATEWAY_ADDRESS || 'http://localhost:8080',
```

Maybe that would allow us to remove the overrides for `ZEEBE_REST_GATEWAY_ADDRESS`.

Reply:
My concern with changing this default is that I don't know 100% that its not in use anymore so for now I'll leave it at that

Reply:
We should be able to track that down by usage of `yarn run test:e2e` ? I also see the `8086` port only being used in the operate e2e workflow, so I would argue it's safe to change to the new default port `8080` indeed.

Reply:
I agree, port `8086` was only used by the the zeebe service container [here](https://github.com/camunda/camunda/pull/27485/files#diff-d046a4c1810127d86e1a7d99c5953d3bcbf91529d672c7fc4a6fa98236b4bdb1L54-L64). Since this is removed now, we should change the default to `8080`.